src/features/cartographie/components/lieux-mediation-numerique-details/lieux-mediation-numerique-details-print.component.html

```diff
@@ -125,9 +125,9 @@ <h1 class="d-none d-print-block h4 fw-bold">{{ lieuMediationNumerique.nom }}</h1
         class="w-25"
         *ngIf="brandConfiguration.logoAvecTexte; else brandLogo"
         [src]="brandConfiguration.logoAvecTexte"
-        alt="" />
+        alt="Logo secondaire" />
```

Comment:
```suggestion
        alt="" />
```
Il faut mieux ne rien mettre que de mettre à tout pris un texte alternatif si l'image n'apporte pas d'information au contenu.
(En plus que là il s'agit d'un rendu pour imprimer ;))